Documentation/Testing/CoreTesting.rst

```diff
@@ -69,37 +66,37 @@ system. Executing the basic core unit test suite boils down to:
     Build/Scripts/runTests.sh
 
 That's it. You just executed the entire unit test suite. How long did test execution take on your side?
-Your local environment is cool if you are below thirty seconds ... ok, let's say two minutes on mac. After
-initial core clone and a composer install, other parts of this chapter are about different permutations of
-calling `runTests.sh` only.
+initial core clone and a composer install, other parts of this chapter are about different permutations of.
+Now that we have examined the initial core clone and a composer install process, we will now look at the
```

Comment:
- Is something missing here? After "How long did test execution take on your side?"
- remove second "now": "Now that we have examined ..., we will now look at."

Reply:
Fixed :)